packages/workspace/src/generators/ci-workflow/schema.json

```diff
@@ -31,7 +31,8 @@
       },
       "default": "CI",
       "x-prompt": "How should we name your workflow?",
-      "pattern": "^[a-zA-Z].*$"
+      "pattern": "^[a-zA-Z].*$",
+      "x-priority": "important"
```

Comment:
Is it not `required`?

Reply:
There's a default value of `CI` that's used when you don't provide a `name`... I guess we could remove that and make `name` required but then again sometimes folks might want to create a workflow and don't care what it's called.